File: engine/model/src/main/scala/net/server/channel/packet/ring/OpenWeddingInvitationPacket.scala
package net.server.channel.packet.ring

class OpenWeddingInvitationPacket(private var _mode: Byte, private var _slot: Byte, private var _invitationId: Int) extends BaseRingPacket(_mode = _mode) {
  def slot: Byte = _slot

  def invitationId: Int = _invitationId
}

File: engine/engine-model/src/main/java/net/server/channel/packet/ring/OpenWeddingInvitationPacket.java
package net.server.channel.packet.ring;

public class OpenWeddingInvitationPacket extends BaseRingPacket {
   private final Byte slot;

   private final Integer invitationId;

   public OpenWeddingInvitationPacket(Byte mode, Byte slot, Integer invitationId) {
      super(mode);
      this.slot = slot;
      this.invitationId = invitationId;
   }

   public Byte slot() {
      return slot;
   }

   public Integer invitationId() {
      return invitationId;
   }
}
